.fides/db_dataset.yml

```diff
@@ -6,1085 +6,731 @@ dataset:
     - name: organization_fides_key
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: rights
       description: JSON structure containing a strategy and optional values for detailing
         data subject rights available
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: tags
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: updated_at
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: is_default
       data_categories:
         - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: active
       data_categories:
         - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: version_added
       data_categories:
         - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: version_deprecated
       data_categories:
         - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: replaced_by
       data_categories:
         - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
   - name: ctl_data_uses
-    data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     fields:
     - name: created_at
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: description
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: fides_key
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
     - name: id
       data_categories:
       - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
-    - name: legal_basis
-      data_categories:
-      - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
-    - name: legitimate_interest
-      description: Boolean value denoting whether or not the data use is marked as
-        a legitimate interest
-      data_categories:
-      - system.operations
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
-    - name: legitimate_interest_impact_assessment
```

Comment:
Nice, did you remember to go into this file and remove this or was it automatically flagged somewhere? I've felt this file only gets flagged if it's missing values not if there are too many

Reply:
i think you're right on that behavior. pretty sure i found the references just via a global string search on the codebase 🤷

Reply:
so, definitely error prone, if i've missed something!